### DISCUSSION TITLE
Add another amazing Repo to the list

### DISCUSSION THREAD

**Discussion by s-e-t-h** [alexbelgium/hassio-addons#1108] [Ideas]
Hi @alexbelgium 

Thank you so much for this repo and your work on the Omada addon situation.

My friend created this repo for the Homarr dashboard and it is great!
You can see it here: https://github.com/Wiggen94/ha-homarr-server

I think many others will like it and I can put you in contact with him if you need any more info.

Thanks!

**Comment by alexbelgium** [comment]
Hi indeed looks nice! What would you like actually from part that I put a link on my repo ? The best is that people know about it then install his own repo on their systems so he can have the visibility of his work and manage potential issues and questions from his own repo

**Reply by s-e-t-h** [reply]
Yes, that's perfect.

I just love my dashboard and would like to share the good news :)

**Reply by alexbelgium** [reply]
I've put a note here for repositories recommendations : https://github.com/alexbelgium/hassio-addons/issues/1112

**Reply by s-e-t-h** [reply]
Thanks, Alex!
